### PR TITLE
Enhancement: Switch to using GitHub actions

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] switches to using GitHub Actions instead of Travis CI

Related to #13.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
-# Recursion Context
+# sebastian/recursion-context
```
Comment (localheinz) [inline README.md:1]: Since touching this file, I took the freedom to streamline the content a bit.

Let me know if you would prefer to revert these changes.

**Comment by localheinz** [comment]
Thank you, @sebastianbergmann!

**Comment by sebastianbergmann** [comment]
* https://github.com/sebastianbergmann/recursion-context/commit/b671fd1c98262f165e8d67a8a96d09bc0ea026dd/checks?check_suite_id=411704260
* https://github.com/sebastianbergmann/recursion-context/runs/400595687
